charts/drupal/values.yaml

```diff
@@ -791,5 +791,10 @@ signalsciences:
 # Logging configurations for services that support them.
 # BETA feature, currently only supported by drupal-cron.
 logging:
-  # Currently supported formats are "default" and "json".
-  format: default
+  # Currently supported formats are:
+  # - "plain"
+  #   - output everything as it runs in real time.
+  # - "json" (default)
+  #   - output everything as it runs in real time.
+  #   - create additional overview-log-entry at the end including the cron job output.
+  format: json
```

Comment:
The changing of default value will still need a bit of discussion, let's do that on Slack.